builtin/mainmenu/init.lua

```diff
@@ -24,6 +24,15 @@ mt_color_dark_green = "#25C191"
 
 local menupath = core.get_mainmenu_path()
 local basepath = core.get_builtin_path()
+local menustylesetting = core.settings:get("main_menu_style")
+local menustyle = menustylesetting
```

Comment:
`local menustyle = core.settings:get("main_menu_style")`, superfluous `menustylesetting`

Reply:
If `menustylesetting` is set to "default", then `menustyle` depends on the platform. Should I just use `menustyle` and then just redefine it?